src/helpers/spotify_manager.ts

```diff
@@ -198,26 +204,47 @@ export default class SpotifyManager {
                 }
             }
 
-            const result = (await dbContext
+            // handle songs with brackets in name, consider all components separately
+            const songNameBracketComponents = song.name.split("(");
+            const songNames = [songNameBracketComponents[0], song.name];
```

Comment:
Does the song name get added to this twice if there isn't a `(` in the name? If so, you could move pushing `songNameBracketComponents[0]` on the `songNames` array into the if statement, so we end up doing less `orWhere`